### DISCUSSION TITLE
Now Auth0 auth now redirects me to login page instead of let me in

### DISCUSSION THREAD

**Discussion by gusmiguens** [cypress-io/cypress#22640] [Plugins]
Hi, I was using this library. https://github.com/sir-dunxalot/cypress-nextjs-auth0 and suddenly it's not working. And let me be more specific: SUDDENLY. There were no changes neither on our side nor auth0, the tokens and appSession are being generated correctly, more over I can manually login with the same user. BUT when I run the tests what I was doing was:

1. Get the token and set the cookie within this library
2. Use that to login directly to my up avoiding auth0 UI.
3. cy visit myrURL/myMenu

Now when I do cy visit I'm being redirect with the code 307 to the login page of Auth0.

Does anyone know what is going on?

Thanks in advance

**Comment by fjpedrosa** [comment]
Is this happening by executing tests in browser, headless or both modes?
 
I implemented a custom command to execute the login [as explained here](https://medium.com/front-end-weekly/how-to-authenticate-auth0-users-in-cypress-e2e-tests-of-angular-1e021a9a7f76#id_token=eyJhbGciOiJSUzI1NiIsImtpZCI6Ijk1NTEwNGEzN2ZhOTAzZWQ4MGM1NzE0NWVjOWU4M2VkYjI5YjBjNDUiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL2FjY291bnRzLmdvb2dsZS5jb20iLCJuYmYiOjE2NjYwMjQ4NjEsImF1ZCI6IjIxNjI5NjAzNTgzNC1rMWs2cWUwNjBzMnRwMmEyamFtNGxqZGNtczAwc3R0Zy5hcHBzLmdvb2dsZXVzZXJjb250ZW50LmNvbSIsInN1YiI6IjExMzYxMDkwNzI1MjI5NDgyNjkyMyIsImVtYWlsIjoiYm9rZWF1c3NAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF6cCI6IjIxNjI5NjAzNTgzNC1rMWs2cWUwNjBzMnRwMmEyamFtNGxqZGNtczAwc3R0Zy5hcHBzLmdvb2dsZXVzZXJjb250ZW50LmNvbSIsIm5hbWUiOiJKYXZpZXIgUGVkcm9zYSIsInBpY3R1cmUiOiJodHRwczovL2xoMy5nb29nbGV1c2VyY29udGVudC5jb20vYS9BTG01d3UxSUFmVDRBa3ViRThWVmk2RTdqb0MtUU9ScWlLMlNTczhpeXRoNFpnPXM5Ni1jIiwiZ2l2ZW5fbmFtZSI6IkphdmllciIsImZhbWlseV9uYW1lIjoiUGVkcm9zYSIsImlhdCI6MTY2NjAyNTE2MSwiZXhwIjoxNjY2MDI4NzYxLCJqdGkiOiI3NWViYTMwZTY2ZTk0Nzk3YzM5ZGMyODVkMzFmOGE1NGQ4YmI5MGRhIn0.pq5n0ublsFm7SVM04vwH1KBFr0Gogs2ZxHCqHEEnCIPDqOk7jCndtlxzwgF--TwLxrEzO3Fgx77GnwBkmByMSKSkFSXal7VMUFyUHrVFMQJn-QvGUkpEigKGBw4wfTOggdEtbjwoInkfByTrvYajS67liWdY5uHI9gkqC5aIO9MfSEc5fSl5njuPgDFtSIf1Ppp3VaDM8fGGaBv9g8snaGxzG4kDpanh49HH5SkSc1UmkB6Ibx1HZg2w5sy92YtDh7hB8j66DI3wuBznNPmbw40NsUKoJiX-ow26IPgE6LUbvTcU2aofXt1WiAzGH9YLz2Wvv4xDSK8WO5Lnv0uznA), but with react instead of angular, storing the token in localstorage. In my case, everything works fine when launching the browser (I use Chrome) but in headless mode is not working, even I set chrome as browser.
